integration/Dockerfile

```diff
@@ -16,4 +16,4 @@ RUN mv ./kubectl /usr/local/bin
 
 COPY bin/kubectl-gadget /bin/kubectl-gadget
 COPY --from=build_base /build/gadget-integration.test /bin/gadget-integration.test
-CMD ["/bin/sh", "-c", "gadget-integration.test -test.v -integration"]
+CMD ["/bin/sh", "-c", "gadget-integration.test -test.v -integration -github-ci -test.timeout 20m"]
```

Comment:
It's not related to your changes but I think we should remove `-test.v`, otherwise all `t.Logf` calls will be printed generating a lot of noise. Please also consider that those are printed as errors (look for `##[error]` in the raw logs), it creates a lot of confusion.

Reply:
I totally agree with you, I really do not understand why `Logf()` prints to `stderr`, I am always confused when I check the CI seeing so much errors...

Reply:
I think the purpose of those logs is to only be printed if the test fails, then it makes sense those are printed to stderr.